scrapy_zyte_api/handler.py

```diff
@@ -218,10 +218,12 @@ async def _download_request(
                 retrying=retrying,
             )
         except RequestError as er:
+            request_id = er.headers.get("request-id")
             error_detail = (er.parsed.data or {}).get("detail", er.message)
```

Comment:
Should we include request-id to the python-zyte-api error message instead?

Reply:
Something like this? https://github.com/zytedata/python-zyte-api/pull/52